packages/r3broot/package.py

```diff
@@ -0,0 +1,40 @@
+# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
+#   Spack Project Developers. See the top-level COPYRIGHT file for details.
+# Copyright 2020 GSI Helmholtz Centre for Heavy Ion Research GmbH,
+#   Darmstadt, Germany
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class R3broot(CMakePackage):
+    git = 'https://github.com/R3BRootGroup/R3BRoot'
+    homepage = 'https://github.com/R3BRootGroup/R3BRoot'
+
+    version('develop', branch='dev')
+
+    resource(name='macros', when='@develop',
+             git='https://github.com/R3BRootGroup/macros',
+             branch='dev')
+
+    depends_on('fairroot')
+
+    depends_on('fairlogger')
+    depends_on('geant3')
+    depends_on('geant4~threads')
+    depends_on('geant4_vmc')
+    depends_on('pythia6')
+    depends_on('pythia8')
+
+    patch('fairlogger_include.patch')
+
+    def build(self, spec, prefix):
+        super(R3broot, self).build(spec, prefix)
```

Comment:
on Debian 8 with gcc 4.9.2 cxx flag `-std=c++11` is missing. compilation fails

Reply:
`-DCMAKE_CXX_STANDARD=11` will add it

Reply:
Added `cmake_args()`. Let's see.